### PR TITLE
2.5.0 release notes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## stable-1.16-2.5.0, stable-1.17-2.5.0, stable-1.18-2.5.0
+
+* Cert-manager
+  - use `delete` upgrade-strategy.
+* Elasticsearch
```
Comment (samvantran) [inline RELEASE_NOTES.md:7]: This should not be in the release. It's the PR we reverted. Can we turn off the automatic update for elasticsearch?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+  - Fix plugin install initcontainer which would fail if plugin already exists. This happens when Node reboots and keeps emptyDir, or if elasticsearch image already contains plugin
+  - In private environments where we replicate all the images, the test image cannot be pulled due it misses imagePullSecrets ([#497](https://github.com/mesosphere/kubernetes-base-addons/pull/497), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+* Fluent-bit:
+  - bump the fluent-bit app version to 1.5.6
+    - aws: utils: fix mem leak in flb_imds_request
+    - fix double free when destroying connections if the endpoint in unavailable
+    - remove noisy error introduced in v1.5.5
+    - fix deletion of pending connections in the destroy_queue ([#538](https://github.com/mesosphere/kubernetes-base-addons/pull/538), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+  - changes the update strategy to `delete`. ([#574](https://github.com/mesosphere/kubernetes-base-addons/pull/574), [@dkoshkin](https://github.com/dkoshkin))
+  - Upgrades fluent-bit to v1.5.7. See https://fluentbit.io/announcements/v1.5.7.
```
Comment (samvantran) [inline RELEASE_NOTES.md:18]: Should probably drop or combine the 1.5.6 notes into 1.5.7

File: RELEASE_NOTES.md (hunk 1 above)
Comment (samvantran) [reply RELEASE_NOTES.md:7]: Looking at milestone/2.5 it doesn't seem like elasticsearch is actually in there https://github.com/mesosphere/kubernetes-base-addons/milestone/7?closed=1

but other addons are like opsportal, nvidia and they aren't shown here

**Review by samvantran — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 2 above)
Comment (joejulian) [reply RELEASE_NOTES.md:18]: I don't know... putting on my sysadmin hat, that wouldn't bother me in the least.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (joejulian) [reply RELEASE_NOTES.md:7]: Opsportal:
image

Nvidia:
image

**Review by branden — APPROVED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (samvantran) [reply RELEASE_NOTES.md]: The Nvidia PR lists a COPS ticket that it resolves. Surely that should contain a release note as it directly fixes a customer problem?

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+  - Adds chart value `podLabels`.
+  - Fix fluentbit configuration to unblock output buffer. ([#590](https://github.com/mesosphere/kubernetes-base-addons/pull/590), [@branden](https://github.com/branden))
+* Kibana
```
Comment (samvantran) [inline RELEASE_NOTES.md:17]: how about this?

```suggestion
* Nvidia
  - No need to specify nvidia-driver tag when using the default AMI for GPU nodes.
* Kibana
```

**Review by samvantran — APPROVED** [review]
Approving ahead of my suggested fix

**Pull request merged**